app/controllers/boxes_controller.rb

```diff
@@ -1,5 +1,5 @@
 class BoxesController < ApplicationController
-  before_action :load_box, except: %i[index new create bulk_destroy print inventory]
+  before_action :load_box, only: %i[show delete destroy]
```

Comment:
```suggestion
  before_action :load_box, only: %i[show destroy]
```
No need for `delete`, that was just a typo from me.